pages/api/hunks.ts

```diff
@@ -0,0 +1,14 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { saveHunk } from '../../utils/db/relevance';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const hunkjson = req.body;
+  console.log("Publishing message: ", hunkjson);
```

Comment:
We don't merge `console.log`s into main branch.
When a log is only there for development purposes, put a `console.log` there. If you want the logs to be there in production forever, use the appropriate logging level: `console.debug`, `console.info`, `console.warn` or `console.error`.